DiyBlog/settings.py

```diff
@@ -13,6 +13,8 @@
 import os
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+from DiyBlog.AppCredentials import SOCIAL_AUTH_GOOGLE_KEY, SOCIAL_AUTH_GOOGLE_SECRET
```

Comment:
This is just a note: normally we read such kind of secret configuration variables from environment variable. We can refactor that in later.